src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/DashboardTopBanner/DashboardTopBannerContent.tsx

```diff
@@ -223,7 +223,8 @@ export const DashboardTopBannerContent = (props: DashboardTopBannerProps) => {
               )}
             </p>
             <div className={styles.cta}>
-              {typeof props.totalNumberOfPerformedScans === "undefined" ||
+              {!props.enabledFeatureFlags.includes("MonitorPlus") ||
```

Comment:
I think another name might be more appropriate? Maybe something like `ForceWaitlist`.. `MonitorPlus` might be too generic/confusing